app/src/components/report/ReportEdit.tsx

```diff
@@ -0,0 +1,209 @@
+                    onChange={(event) => updateReport({ emailAddress: event.target.value })}
+                />
+            </Form.Group>
+
+            <Form.Group className="mb-3" controlId="EditReport.PhoneNumber">
+                <Form.Label>Phone Number</Form.Label>
+                <Form.Control
+                    type="tel"
+                    placeholder="Please enter your phone number"
+                    defaultValue={report?.phoneNumber}
+                    onChange={(event) => updateReport({ phoneNumber: event.target.value })}
+                />
+            </Form.Group>
+
+            <Form.Group className="mb-3" controlId="EditReport.ReportCategory">
+                <Form.Label>Report Category</Form.Label>
+                {[
+                    'Illegal Dumping',
+                    'Clogged Storm Drain',
+                    'Potholes',
+                    'Graffiti',
+                    'Street Light Outage',
+                    'Sidewalk Damage',
+                    'Traffic Signal Malfunction',
+                    'Abandoned Vehicles',
+                    'Noise Complaint',
+                    'Other',
+                ].map((category) => (
+                    <Form.Check
+                        type="checkbox"
+                        label={category}
+                        checked={report?.reportCategory.includes(category)}
+                        onChange={(event) => {
+                            const newCategories = [...report!.reportCategory];
+                            if (event.target.checked) {
+                                newCategories.push(category);
+                            } else {
+                                newCategories.splice(newCategories.indexOf(category), 1);
+                            }
+                            updateReport({ reportCategory: newCategories });
+                        }}
+                    />
+                ))}
+            </Form.Group>
+
+            <Form.Label>Issue Location</Form.Label>
+
+            <Form.Group className="mb-3" controlId="EditReport.Address">
+                <Form.Label>Address</Form.Label>
+                <Form.Control
+                    type="text"
+                    placeholder="Please enter your address"
+                    defaultValue={report?.address}
+                    onChange={(event) => updateReport({ address: event.target.value })}
+                />
+            </Form.Group>
+
+            <Form.Group className="mb-3" controlId="EditReport.GpsCoordinates">
+                <Form.Label>GPS Coordinates</Form.Label>
+                <Form.Control
+                    type="text"
+                    placeholder="Please enter your GPS Coordinates"
+                    defaultValue={report?.gpsCoordinates}
+                    onChange={(event) => updateReport({ gpsCoordinates: event.target.value })}
+                />
+            </Form.Group>
```

Comment:
Should we remove this and stick with address for now? It's confusing for the customer to have both. As a separate item, we can make this form accept either an address (type in) or GPS coordinates (map-picker)